x-pack/plugins/fleet/server/collectors/agent_policies.ts

```diff
@@ -26,29 +49,45 @@ export const getAgentPoliciesUsage = async (
   abortController: AbortController
 ): Promise<AgentPoliciesUsage> => {
   try {
-    const res = await esClient.search(
+    const res = await esClient.search<{}, AgentPoliciesUsageAggs>(
```

Comment:
should we add a `ignore_unavailable: true` as it's possible the index do not exists if there is no agent policy

Reply:
Done in https://github.com/elastic/kibana/pull/149707/commits/23a4bf08a5514e42f630a384dca0efdcec2cd636